docs/setup-aws-lambda.asciidoc

```diff
@@ -1,85 +1,68 @@
 [[aws-lambda]]
-=== Get started with Lambda
+=== Monitoring AWS Lambda Java Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
-
-NOTE: In order to get the full AWS Lambda tracing capabilities, use with APM Server 7.16 or higher. Using
-older versions provides most value, but some relevant metadata fields will not be indexed.
+The Java APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
 [float]
-[[aws-lambda-runtimes]]
-==== Supported runtimes
-
-AWS Lambda provides multiple https://docs.aws.amazon.com/lambda/latest/dg/java-image.html[JVM base images], only the ones that support the `AWS_LAMBDA_EXEC_WRAPPER` environment variables
-are supported out of the box.
-
-Running with unsupported images is still possible but requires providing agent configuration through environment variables
-explicitly.
+[[aws-lambda-java-prerequisites]]
+==== Prerequisites
 
-|===
-|Tags |Java Runtime |Operating System|Supported
+. You need an APM Server to send APM data to. Follow the {apm-guide-ref}/apm-quick-start.html[APM Quick start] if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {ecloud} in the same AWS region as your AWS Lambda functions.
+. Make sure you are using one of the supported AWS Lambda Java runtimes:
++
+--
+include::supported-technologies.asciidoc[tag=aws-lambda-runtimes]
+--
```

Comment:
Nice! I like this. FYI I moved the table include into a [block element](https://docs.asciidoctor.org/asciidoc/latest/blocks/). This allows us to nest it under the accompanying bullet point with a `+`.